### PR TITLE
MBS-10679 / MBS-10680: WS detail tab improvements

### PR DESCRIPTION
**Pull request by reosarevok**

### CODE REVIEW

File: root/entity/Details.js [hunk 1]
```diff
@@ -21,10 +21,11 @@ type Props = {
...
 
-const XMLLink = ({
+const WSLink = ({
   entityGid,
   entityProperties,
   entityType,
+  isJson,
   isSecureConnection,
 }) => {
   const xmlInc = [];
```
Comment (mwiencek) [inline root/entity/Details.js]: Should probably rename this to just `inc` too :P

File: root/entity/Details.js [hunk 2]
```diff
@@ -89,10 +91,23 @@ const Details = ({
         <tr>
           <th>{l('XML:')}</th>
           <td>
-            <XMLLink
+            <WSLink
               entityGid={entity.gid}
               entityProperties={entityProperties}
               entityType={entityType}
+              isJson={false}
```
Comment (yvanzo) [inline root/entity/Details.js]: Probably unneeded? Or if the idea is to highlight this is XML, maybe replace `isJson` with `format` and use `format={'xml'}` instead here?

Comment (reosarevok) [reply root/entity/Details.js]: IIRC Flow complained without it? Can check when I get home

File: root/entity/Details.js [hunk 3]
```diff
@@ -16,28 +16,37 @@ import EntityLink from '../static/scripts/common/components/EntityLink';
 import chooseLayoutComponent from '../utility/chooseLayoutComponent';
 import formatUserDate from '../utility/formatUserDate';
 
-type Props = {
+type WSLinkProps = {
+  +entityGid: string,
+  +entityProperties: {[property: string]: any},
```
Comment (reosarevok) [inline root/entity/Details.js]: We don't have a flow type for this (ENTITIES[type]) yet, right? Should we have a more detailed one?

**Pull request merged**